projects/examples/src/components/datagrid/datagrid-row-icon.example.component.html

```diff
@@ -0,0 +1,5 @@
+<p>
+    Shows that when the data within the grid is reloaded but the trackBy value stays the same, the DOM is not
+    re-rendered. Thus, any row within the datagrid maintains its state.
```

Comment:
Please explain that otherwise, the modal would be closed when re-rendering the rows...